src/core/broadcaster.ts

```diff
@@ -58,7 +59,10 @@ export module Core {
      * @returns {Broadcaster} The calling Broadcaster
      */
     public broadcast(...args: any[]) {
-      this._key2callback.values().forEach((callback) => callback(this._listenable, args));
+      args.unshift(this._listenable);
```

Comment:
that's fine. It's generated locally (see [here](https://github.com/palantir/plottable/pull/1638/files#diff-cf6e1d12d6a4046561f8ef977741d1a1R1103)).